chat-widget/src/common/utils.ts

```diff
@@ -392,7 +392,8 @@ export const getConversationDetailsCall = async (facadeChatSDK: FacadeChatSDK, l
     try {
         TelemetryHelper.logSDKEvent(LogLevel.INFO, {
             Event: TelemetryEvent.GetConversationDetailsCallStarted,
-            Description: "Conversation details call started"
+            Description: "Conversation details call started",
+            LogToAppInsights: false
```

Comment:
LogToAppInsights should be optional and only logged when specifically is set to true, otherwise do not log it

this will reduce the size of the PR